gwpy/timeseries/timeseries.py

```diff
@@ -289,6 +289,8 @@ def psd(self, fftlength=None, overlap=None, window='hann',
         - ``'bartlett'`` : a mean average of non-overlapping periodograms
         - ``'median'`` : a median average of overlapping periodograms
         - ``'welch'`` : a mean average of overlapping periodograms
+
+        If ``method=None`` is passed, this will default to 'welch'.
```

Comment:
Instead of this, could you just modify the `method` parameter docstring above similar to what is done in `gwpy.astro`?

```
        method : `str`, optional
            FFT-averaging method (default: ``'welch'``), see *Notes* for more details
```
or similar

Reply:
@duncanmmacleod, this is done in the latest commit